infra/k8s/base/configmap/init.py

```diff
@@ -639,6 +639,8 @@ def cmd_zilliqa_daemon(args, recovery=False, resume=False):
 
     start_sh = [
         '#!/bin/bash',
+        'ulimit -c unlimited',
+        'sysctl -w kernel.core_pattern=core.%p.%s.%c.%d.%P',
```

Comment:
should this be here? The default location for core dumps seemed to be apport, the crash reporting manager. Only problem is, it doesn't exist in the container so dumps get lost.